flux:
  camera_info: "/zed2i_c1/zed_node_c1/rgb/camera_info"
  imu:         "/imu/data"
  depth_info:  "/zed2i_c1/zed_node_c1/depth/depth_info"
  pointcloud:  "/zed2i_c1/zed_node_c1/point_cloud/cloud_registered"
  rgb_image:   "/zed2i_c1/zed_node_c1/rgb/image_rect_color"
  depth_image: "/zed2i_c1/zed_node_c1/depth/depth_registered"
dmc:
  #empty
pub: "/camera_path_width"